src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,30 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  selectedTodo: Todo,
+  setSelectedTodo: (todo: Todo | null) => void,
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo,
+  setSelectedTodo,
+}) => {
+  const [currUser, setCurrUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    getUser(selectedTodo.userId)
```

Comment:
You should handle the case when the API request fails, for example by adding a catch block to handle errors.